UI/GameSettingsScreen.cpp

```diff
@@ -472,6 +472,8 @@ void GameSettingsScreen::CreateViews() {
 
 #ifdef _WIN32
 	networkingSettings->Add(new PopupTextInputChoice(&g_Config.proAdhocServer, n->T("Change proAdhocServer Address"), "", 255, screenManager()));
+#elif defined(ANDROID)
+	networkingSettings->Add(new ChoiceWithValueDisplay(&g_Config.proAdhocServer, sy->T("Change proAdhocServer Address"), nullptr))->OnClick.Handle(this, &GameSettingsScreen::OnChangeproAdhocServerAddress);
```

Comment:
Just leave the existing one?

```c++
networkingSettings->Add(new ChoiceWithValueDisplay(&g_Config.proAdhocServer, n->T("Change proAdhocServer Address"), nullptr))->OnClick.Handle(this, &GameSettingsScreen::OnChangeproAdhocServerAddress);
```

We want to use `n->` and the lines are the same.

-[Unknown]